man/zram-generator.md

```diff
@@ -0,0 +1,57 @@
+zram-generator(8) -- Systemd unit generator for zram swap devices
+=================================================================
+
+## SYNOPSIS
+
+`/lib/systemd/system-generators/zram-generator` `TARGET_DIR` [*2RGET_DIR* *3RGET_DIR*]<br />
+`/lib/systemd/system-generators/zram-generator` --setup-device `DEVICE`<br />
+`/lib/systemd/system-generators/zram-generator` --reset-device `DEVICE`
```

Comment:
`/usr/lib` please. Both versions are equivalent on modern systems, but `/usr/lib` is the canonical location.